### PR TITLE
feat: WEB-496 Update maxwidth for content images

### PR DESCRIPTION
**Pull request by carlagn**
Fixes #WEB-496

### CODE REVIEW

File: src/theme/styles.module.scss [hunk 1]
```diff
@@ -41,3 +41,11 @@
     }
   }
 }
+
+.img {
+  max-width: 700px !important;
```
Comment (mhessdev) [inline src/theme/styles.module.scss]: gross !important but ok :P

File: src/theme/MDXComponents.tsx [hunk 2]
```diff
@@ -137,6 +139,12 @@ const StyledLink: React.FC<React.PropsWithChildren<ComponentProps<"a">>> = ({
     );
 };
 
+const Image: React.FC<React.PropsWithChildren<ComponentProps<"img">>> = ({
+  ...props
+}) => {
+  return <img {...props} className={clsx(props.className, styles.img)} />
```
Comment (mhessdev) [inline src/theme/MDXComponents.tsx:133]: Shouldn't this be the proper Image component?

**Review by mhessdev — DISMISSED** [review]

File: src/theme/styles.module.scss (hunk 1 above)
Comment (carlagn) [reply src/theme/styles.module.scss]: Very 🫠 but it was either this or ejecting a whole new component from the layouts

File: src/theme/MDXComponents.tsx (hunk 2 above)
Comment (carlagn) [reply src/theme/MDXComponents.tsx:133]: I thought of it but it'd be better to keep it in a separate PR/issue.

**Review by mhessdev — DISMISSED** [review]

**Review by mhessdev — APPROVED** [review]

**Pull request merged**